modules/CoreLibrary/Sources/DependencyInjection/Bootstrap.swift

```diff
@@ -0,0 +1,26 @@
+import Foundation
+
+struct Environment {
+    var globalServiceLocator: ServiceLocatorInterface
```

Comment:
Would like to discuss on our next session, just to see if I understand well the purpose of the Environment struct for bootstrap the layers 🙂🤘🏻